app/Repositories/SignupRepository.php

```diff
@@ -43,6 +45,10 @@ public function create($data)
             $signup->save(['timestamps' => false]);
```

Comment:
@sbsmith86 no worries! this saves the signup with the timestamps that were passed with the rest of the data, if any were. so for the migration we are able to keep the original timestamps.